README.md

```diff
@@ -348,3 +348,64 @@ When you want to add an environment variable, make sure to modify the following
 - `.env.local`
   - This file is used by test runners.
   - Add each new environment variable to the file.
+
+# Frontend API Access via Proxy
+
+This guide explains how to set up the nginx-based reverse proxy for accessing private API Gateway endpoints during development.
+The proxy setup creates a secure tunnel to private API Gateway endpoints:
+
+```
+Frontend App → localhost:8888 → Session Manager → nginx → VPC Endpoint → API Gateway
+```
+
+## Prerequisites
+
+- AWS CLI configured
+- Session Manager plugin: `brew install session-manager-plugin`
+- Node.js and yarn installed
+- Access to `di-authentication-development-admin` AWS profile
```

Comment:
This is a legacy role so will soon be removed.  Can it depend on a standing permission role instead?

Also this role is in the old account but the api runs in the new account, is that correct?